### ISSUE TITLE
Make HOCON config ToString() method more readable

### ISSUE THREAD

**Issue by Horusiath** [akkadotnet/akka.net#1703]
Right now `HoconConfig` `ToString` method produces string containing only key-vals of the current config. Since structure of config objects often heavily relies of fallback mechanism, in order to get all current (overriden) key-vals in current config section is very exhausting. 

Therefore we could change current ToString behavior (or expose its version in form of `ToString(string format)`) that could return all of the current key-values for the hocon config. Example, given two hocon configs A and B with set in fallback relation:

```hocon
# A
some.path {
  key11 = "value11"
  key12 = "value12"
}

# B
some.path {
  key11 = "value21"
  key22 = "value22"
}
``` 

using `configA.WithFallback(configB).ToString(indentation: true)` would print current key-values of A, followed by key-values being fallback from B:

```hocon
some.path {
  key11 = "value11" 
  key12 = "value12"
  key22 = "value22"
}
```

**Comment by mukulsinghsaini** [comment]
Hi Horusiath,
I have implemented this method. It gives the current key-values merged with key-values from fallback.
It can work on multiple fallback chains like` configA.WithFallback(configB).WithFallback(configC)...`

I have used the method signature
`public string ToString(Boolean includeFallback)`

I want to confirm if its ok to use this method signature or I need to improve it before I send a pull request.Thanks.
